Makefile

```diff
@@ -1,6 +1,6 @@
 # Variables are documented in common/build.sh.
 BASE_IMAGE_NAME = httpd
-VERSIONS = 2.4 2.4-micro
+VERSIONS = 2.4-micro 2.4
```

Comment:
Could you please elaborate more on how the order of mentioned versions fixes the situation? I think I do not understand it entirely.

Reply:
In case of `2.4` is not at the end then the latest imagestreams are skipped.

```bash
Image version 2.4 is not latest, skipping ct_check_latest_imagestreams
Already on project "default" on server "https://api.core-serv-ocp.hosted.psi.rdu2.redhat.com:6443".
 [PASSED] for 'openshift-remote-cluster' test_latest_imagestreams (00:00:00)
```

In case of `2.4` is at the end then the output is:
```
Version to check is 2.4.
Checking file imagestreams/httpd-centos.json.
check_version: Compare tags['name']:'latest' against version:'2.4'
check_version: Compare tags['name']:'2.4-fc' against version:'2.4'
check_version: Compare tags['name']:'2.4-micro-fc' against version:'2.4'
check_version: Compare tags['name']:'2.4-ubi9' against version:'2.4'
check_version: Compare tags['name']:'2.4-micro-c9s' against version:'2.4'
check_version: Compare tags['name']:'2.4-ubi8' against version:'2.4'
check_version: Compare tags['name']:'2.4-micro-c8s' against version:'2.4'
check_version: Compare tags['name']:'2.4-el7' against version:'2.4'
check_version: Compare tags['name']:'2.4' against version:'2.4'
check_latest_tag: Compare tags['name']:'latest' against version:'2.4'
Latest tag found.
Checking file imagestreams/httpd-rhel-aarch64.json.
check_version: Compare tags['name']:'latest' against version:'2.4'
check_version: Compare tags['name']:'2.4-ubi9' against version:'2.4'
check_version: Compare tags['name']:'2.4-ubi8' against version:'2.4'
check_version: Compare tags['name']:'2.4-el8' against version:'2.4'
check_latest_tag: Compare tags['name']:'latest' against version:'2.4'
Latest tag found.
Checking file imagestreams/httpd-rhel.json.
check_version: Compare tags['name']:'latest' against version:'2.4'
check_version: Compare tags['name']:'2.4-ubi9' against version:'2.4'
check_version: Compare tags['name']:'2.4-ubi8' against version:'2.4'
check_version: Compare tags['name']:'2.4-el8' against version:'2.4'
check_version: Compare tags['name']:'2.4-el7' against version:'2.4'
check_version: Compare tags['name']:'2.4' against version:'2.4'
check_latest_tag: Compare tags['name']:'latest' against version:'2.4'
Latest tag found.
Imagestreams contains the latest version.
Already on project "default" on server "https://api.core-serv-ocp.hosted.psi.rdu2.redhat.com:6443".
 [PASSED] for 'openshift-remote-cluster' test_latest_imagestreams (00:00:00)

```

micro and minimal images are not present in the imagestreams.

Reply:
The latest versions are taken by https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L1124